python/cudf/cudf/tests/test_datetime.py

```diff
@@ -444,7 +444,6 @@ def test_datetime_has_null_test(data, expected):
     if False in count.keys():
         expected_count = count[False]
 
-    assert_eq(expected, data.has_null_mask)
```

Comment:
Add this assertion back once you readd the attribute?

Reply:
Done. We now have `.has_nulls` and `.nullable` and I've used them throughout